tests/unit/objects/test_personal_access_tokens.py

```diff
@@ -41,15 +42,47 @@ def resp_create_user_personal_access_token():
 
 
 @pytest.fixture
-def resp_personal_access_token(no_content):
-    with responses.RequestsMock(assert_all_requests_are_fired=False) as rsps:
+def resp_list_personal_access_tokens():
+    with responses.RequestsMock() as rsps:
         rsps.add(
             method=responses.GET,
             url=token_url,
             json=[content],
             content_type="application/json",
             status=200,
         )
+        yield rsps
+
+
+@pytest.fixture
+def resp_get_personal_access_token():
+    with responses.RequestsMock() as rsps:
+        rsps.add(
+            method=responses.GET,
+            url=single_token_url,
+            json=content,
+            content_type="application/json",
+            status=200,
+        )
+        yield rsps
+
+
+@pytest.fixture
+def resp_get_personal_access_token_self():
+    with responses.RequestsMock() as rsps:
+        rsps.add(
+            method=responses.GET,
+            url=self_token_url,
+            json=content,
+            content_type="application/json",
+            status=200,
+        )
+        yield rsps
```

Comment:
A bit repetitive here but I think that's ok for tests as we would parametrize only 2 variants.